src/main/java/org/apache/ibatis/builder/xml/mybatis-mapper.xsd

```diff
@@ -108,10 +108,10 @@
   </xs:element>
   <xs:element name="constructor">
     <xs:complexType>
-      <xs:sequence>
-        <xs:element minOccurs="0" maxOccurs="unbounded" ref="idArg"/>
-        <xs:element minOccurs="0" maxOccurs="unbounded" ref="arg"/>
-      </xs:sequence>
+      <xs:choice minOccurs="0" maxOccurs="unbounded">
```

Comment:
Not that your change is wrong, but it would be better to set `minOccurs="1"` to avoid empty `<constructor>` i.e.

```xml
<constructor></constructor>
```

(this is not possible with DTD, AFAIK)